openassessment/templates/openassessmentblock/staff/oa_staff_cancelled.html

```diff
@@ -0,0 +1,22 @@
+{% load i18n %}
+
+<li id="openassessment__staff-assessment" class="openassessment__steps__step step--staff-assessment">
```

Comment:
Can you try using a class instead of an id for all these new templates. It would be good to not introduce any new ids unless we really need them for some reason.